02_plotting-mutation-counts.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "Plotting Gene Mutation Counts"
+author: Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+---
+
+## Introduction
+
+We at the Data Lab really like using notebook documents like [Rmarkdown](https://rmarkdown.rstudio.com) (this document) and [Jupyter](https://jupyter.org) for development and data exploration.
+Notebooks allow mixing of text, code, and results in a single document that is easy to work with interactively.
+This allows much more freedom to include a lot of explanatory text, much like code comments, but with simpler syntax and the ability to include formatting and links more easily through the use of [markdown-formatted text](https://rmarkdown.rstudio.com/lesson-8.html) (or using the new visual editor in RStudio).
+
+When you are ready to share, notebooks like this can easily be converted into a document such as `.html` (web page) or `.docx` (MS Word document) for sharing with others.
```

Comment:
```suggestion
When you are ready to share your results with others, notebooks like this can easily be converted into a document such as `.html` (web page) or `.docx` (MS Word), results, plots, and all.
```